src/core/turn-order.test.ts

```diff
@@ -779,7 +810,13 @@ describe('setActivePlayers', () => {
       const reducer = CreateGameReducer({ game });
       let state = InitializeGame({ game, numPlayers: 3 });
 
-      expect(state.ctx._activePlayersMoveLimit).toEqual({
+      expect(state.ctx._activePlayersMinMoves).toEqual({
+        '0': 1,
+        '1': undefined,
+        '2': undefined,
+      });
```

Comment:
We should use [`toStrictEqual`](https://jestjs.io/docs/expect#tostrictequalvalue) to check the *absence* of the `undefined` keys here:

```suggestion
      expect(state.ctx._activePlayersMinMoves).toStrictEqual({ '0': 1 });
```